bundles/framework/myfeatures/handler/MyFeaturesHandler.js

```diff
@@ -212,6 +237,95 @@ class MyFeaturesHandler extends StateHandler {
         }
     }
 
+    // strip myf_ from the beginning of layerId for db
+    getLayerUUID(layer) {
+        if (!layer?.id?.length > 4) {
+            return null;
+        }
+
+        return layer?.id?.substring(4, layer.id.length) || null;
```

Comment:
On the layer operations the server-side code recognizes this and the full layer id with `myf_`-prefix can be used. I would do that here as well so the frontend code doesn't have to modify the layer id that server uses.